etl/message/attachment.py

```diff
@@ -34,17 +41,14 @@ def extract_attachment(self, attachment: pypff.attachment) -> str:
 def save_attachment(m):
     for a in m.attachments:  # type: pypff.attachment
         logger.info(f"Storing attachment({a.identifier}): {a.name} - {a.size}")
-        hashed_name = self._save_attachment(a)
+        hashed_name = "fixme"
```

Comment:
Is this for testing?

Reply:
self was failing flake8 because it doesn't exist. I don't think this code is functional atm, it was just moved here for later use.